HOWTO.md

```diff
@@ -192,6 +287,50 @@ math.max({a}, {b})                                     *math.load_extension()*
         table: bigger number
 ```
 
+## See
+
+Reference something else.
+
+```lua
+local math = {}
+
+--- Will return the bigger number
+---@param a number: first number
+---@param b number: second number
+---@return number: bigger number
+---@see math.min
+math.max = function(a, b)
+  if a > b then
+    return a
+  end
+  return b
+end
+
+return math
+```
+
+Output:
+
+```
+math.max({a}, {b})                                     *math.load_extension()*
+    Will return the bigger number
+
+
+    Parameters: ~
+        {a} (number)  first number
+        {b} (number)  second number
+
+    Return: ~
+        table: bigger number
+
+    See: ~
+        |x.bye()|
```

Comment:
What is this?

Reply:
Can't i just link to a different help tag like that? I am not that familiar with writing vim help, so i have to check again

Reply:
Oh now i see what you saying, copy paste mistake :laughing: